llvm/benchmarks/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 set(LLVM_LINK_COMPONENTS
   Support)
 
-add_benchmark(DummyYAML DummyYAML.cpp)
+add_benchmark(DummyYAML DummyYAML.cpp PARTIAL_SOURCES_INTENDED)
+add_benchmark(xxhash xxhash.cpp PARTIAL_SOURCES_INTENDED)
```

Comment:
@BertalanD @kirillbobyrev this adds a ninja target `xxhash` which just builds `benchmarks/xxhash`. Is that necessary or even useful? It caused a clash for us downstream when LLVM is included as a subproject into a larger project, because some other code also used `xxhash` as a ninja target name.

Reply:
Sorry, I didn't expect it to create a top-level Ninja target. I would be inclined to rename it to `benchmarkLLVMxxhash` or the like to reduce the chance of collisions. But there's some CMake trick for this that I do not know of. Any ideas?

Reply:
@jayfoad https://github.com/llvm/llvm-project/pull/100172